KiwiLooper/objects/Debug/Create_0.gml

```diff
@@ -36,3 +36,23 @@ if (convention_mode || demo_mode)
 
 // Game debugging values
 lively_show_explode_wire_target = true;
+
+
+Show = function()
+{
+	if (!on)
```

Comment:
do we need a second (!on) check here? if have one here, definitely also put a corresponding one in Hide()